persistence/src/main/java/de/cyface/persistence/Constants.java

```diff
@@ -20,17 +16,15 @@
     /**
      * The file extension of the measurement file which is transmitted on synchronization.
      */
-    public static final String TRANSFER_FILE_EXTENSION = "cyf";
+    public static final String TRANSFER_FILE_EXTENSION = "ccyf";
```

Comment:
I fixed the typo "cyf" instead of "ccyf" as transfer file name extension already previously but this was accidentally done the "hard-coded" way instead of using this constant which was not in use until now.